### PR TITLE
Added checks to ensure that schedules are syncing

### PR DESCRIPTION
**Pull request by kevinkrieger**
... to the 5 sites from superdarn-cssdp using the unison crontab entries of the mrcopy user.

Tested and currently running. See http://superdarn-cssdp.usask.ca/nagios/

### CODE REVIEW

File: configs/monitor/objects/commands.cfg [hunk 1]
```diff
@@ -348,8 +348,14 @@ define command {
 #        both local and remote hosts.
 ################################################################################
 
+# CHeck the Borealis schedule sync - takes one file name for the log file from the
```
Comment (RemingtonRohel) [inline configs/monitor/objects/commands.cfg:351]:
```suggestion
# Check the Borealis schedule sync - takes one file name for the log file from the
```

File: configs/monitor/objects/maxwell.cfg [hunk 2]
```diff
@@ -1,302 +1,293 @@
...
-	servicegroups		autossh
-	check_command		check_nrpe!check_ssh!-a '-p 50822 localhost'
+    use                 generic-service
+    host_name           maxwell
+    service_description Autossh: tcd
+    servicegroups       autossh
+    check_command       check_nrpe!check_ssh!-a '-p 50822 localhost'
 }
-
+<
```
Comment (RemingtonRohel) [inline configs/monitor/objects/maxwell.cfg:61]: Not sure what this is for

File: configs/monitor/objects/maxwell.cfg [hunk 3]
```diff
@@ -1,302 +1,293 @@
...
+    host_name           maxwell
+    service_description Root Partition
+    check_command       check_nrpe!check_disk!-a '-w 10% -c 5% -p / -p /home -p /tempdat /local_data'
 }
 
 # The following services are for monitoring the autossh tunnels to remote computers.
 # The format for each port is 5xyzz:
-# 	x is determined by the radar site
-# 	0 - SAS, 1 - PGR, 2 - CLY, 3 - RKN, 4 - INV
+#     x is determined by the radar site
```
Comment (RemingtonRohel) [inline configs/monitor/objects/maxwell.cfg:39]:
```suggestion
#    x is determined by the radar site
```

File: configs/monitor/objects/maxwell.cfg [hunk 4]
```diff
@@ -1,302 +1,293 @@
...
+#     0 - SAS, 1 - PGR, 2 - CLY, 3 - RKN, 4 - INV
```
Comment (RemingtonRohel) [inline configs/monitor/objects/maxwell.cfg:40]:
```suggestion
#    0 - SAS, 1 - PGR, 2 - CLY, 3 - RKN, 4 - INV
```

File: configs/monitor/objects/maxwell.cfg [hunk 5]
```diff
@@ -1,302 +1,293 @@
...
 #
-# 	y is determined by the site computer
-#	5 - borealis, 6 - dds, 8 - cd
+#     y is determined by the site computer
```
Comment (RemingtonRohel) [inline configs/monitor/objects/maxwell.cfg:42]:
```suggestion
#    y is determined by the site computer
```

File: configs/monitor/objects/maxwell.cfg [hunk 6]
```diff
@@ -1,302 +1,293 @@
...
+#    5 - borealis, 6 - dds, 8 - cd
 # 
-# 	zz is determined by the tunnel
-#	22 - default, 23 - alternate
+#     zz is determined by the tunnel
```
Comment (RemingtonRohel) [inline configs/monitor/objects/maxwell.cfg:45]:
```suggestion
#    zz is determined by the tunnel
```

File: configs/monitor/objects/nagios.cfg [hunk 7]
```diff
@@ -1,1381 +0,0 @@
-##############################################################################
```
Comment (RemingtonRohel) [inline configs/monitor/objects/nagios.cfg:1]: Is this file not needed?

**Review by RemingtonRohel — CHANGES_REQUESTED** [review]
The code suggestions are due to my inconsistencies with whitespace in the autossh comments, so in the sake of cleaning up whitespace I think we fix these last spots and it'll be perfect.

File: configs/monitor/objects/maxwell.cfg (hunk 2 above)
Comment (kevinkrieger) [reply configs/monitor/objects/maxwell.cfg:61]: crap, this is a typo

Comment (kevinkrieger) [reply configs/monitor/objects/maxwell.cfg:61]:
```suggestion

```

File: configs/monitor/objects/nagios.cfg (hunk 7 above)
Comment (kevinkrieger) [reply configs/monitor/objects/nagios.cfg:1]: There's another more up-to-date version in configs/monitor/

**Review by RemingtonRohel — APPROVED** [review]

**Pull request merged**